### PR TITLE
Reposition Post back button outside of Post border

### PR DESCRIPTION
**Pull request by nr431207**
### Centering the back button outside of the post border box.
Screen Shot 2020-06-25 at 2 31 49 PM

_Please be concise and link any related issue(s):_
#804 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by nr431207** [comment]
@ppagarawal let me know if this looks good :)

File: client/src/components/Feed/Post.js [hunk 1]
```diff
@@ -296,12 +296,12 @@ const Post = ({
...
-        {postId && 
-        <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
-        }   
+        </WebModal>  
       </PostCard>
-    </>
+      {postId && 
+        <StyledWizard nav={<WizardFormNav />}></StyledWizard>
+      } 
+    </div>
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:304]: So wait, are you having this inside the post component? I thought we wanted it outside.

File: client/src/components/StepWizard/WizardFormNav.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 const desktopBreakpoint = mq.tablet.narrow.maxWidth;
 
-export const StyledButtonWizard = styled(StepWizard)`
+const StyledWizardNav = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding-left: 12rem;
-  margin: 0 auto 1rem;
+  padding: 0 6rem 1rem;
```
[the remark this answers is not in the document]
Comment (nr431207) [reply client/src/components/StepWizard/WizardFormNav.js:14]: I created this modified style solely for this button, and only using it here, so no it wont :)

File: client/src/components/StepWizard/WizardFormNav.js [hunk 3]
```diff
@@ -30,10 +29,14 @@ export const StyledButtonWizard = styled(StepWizard)`
...
   }
+
+  @media screen and (max-width: ${desktopBreakpoint}) {
+    padding: 0 13rem 1rem;
+  }
 `;
 
 const WizardFormNav = () => (
-  <StyledWizardNav>
+  <StyledWizardNav>
```
[the remark this answers is not in the document]
Comment (nr431207) [reply client/src/components/StepWizard/WizardFormNav.js:39]: will fix!

File: client/src/components/Feed/Post.js (hunk 1 above)
Comment (nr431207) [reply client/src/components/Feed/Post.js:304]: outside the `border` of the post to be specific, but still within the component.

File: client/src/components/StepWizard/StyledWizard.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { mq } from "constants/theme";
 
 const { tablet, desktop } = mq;
 
-const StyledWizard = styled(StepWizard)`
+export const StyledWizard = styled(StepWizard)`
```
[the remark this answers is not in the document]
Comment (nr431207) [reply client/src/components/StepWizard/StyledWizard.js:7]: actually im importing this into Post component.

**Comment by ppagarawal** [comment]
@nr431207 @idjevm Please check the alignment of the button. It appears to be fine for a few posts but not for all.

image

File: client/src/components/StepWizard/StyledWizard.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (idjevm) [reply client/src/components/StepWizard/StyledWizard.js:7]: You are already exporting this component on line L37, so no need to export here on line 7..

File: client/src/components/StepWizard/WizardFormNav.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (idjevm) [reply client/src/components/StepWizard/WizardFormNav.js:14]: Sweet! :)

**Comment by ppagarawal** [comment]
> Any comments on the styling @ppagarawal ? I noticed that there is some mix up coloring happening on active and on hover. https://share.getcloudapp.com/OAuBG8bN
> 
> 

These will be addressed in future releases. These changes on mouse over and click is common across the platform.

File: client/src/components/Feed/Post.js [hunk 5]
```diff
@@ -338,57 +337,59 @@ const Post = ({
...
+              cancelText="Cancel"
+            >
+              <p>Are you sure you want to delete the post?</p>
+            </WebModal>
+          </PostCard>
           {showComments && (
-            <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
-          )}
-        </PostCard>
+              <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:390]: is this some issue with your prettier settings? the formatting seems a bit off here

File: client/src/components/StepWizard/WizardFormNav.js [hunk 6]
```diff
@@ -26,7 +25,7 @@ export const StyledButtonWizard = styled(StepWizard)`
       min-height: 100%;
     }
   }
-
+
```
Comment (idjevm) [inline client/src/components/StepWizard/WizardFormNav.js:28]: remove the 2 white spaces on line 28 - do not remove the line, just the white spaces

**Pull request merged**